src/styles/base/_tables.scss

```diff
@@ -16,8 +16,13 @@ td {
 	}
 }
 
-tr:last-child td,
-tr:last-child th {
+tbody:last-child > tr:last-child td {
```

Comment:
Hmm.. Feel like it has potential to break to me, but if you're happy with it then 👍